src/upnp.h

```diff
@@ -31,7 +31,8 @@ typedef struct {
 	socket_t sock;
 	char external_addr_str[ADDR_MAX_STRING_LEN];
 	char *location_url;
-	int wanipconnection_ver;
+	char *service;
```

Comment:
This should be const to assign string constants.
```suggestion
	const char *service;
```